server/queue.lua

```diff
@@ -1,4 +1,4 @@
-if GetConvar('qbx:enablequeue', 'true') == 'false' then return end
+if GetConvar('qbx:enablequeue', 'true') == 'false' then return false end
```

Comment:
Why is it better to return false?

Reply:
Feels much nicer and more readable to be able to do a simple `if not queue` check than checking if the table is empty or if the required module returns some other arbitrary value.

Reply:
what happens if you return nil? if not queue doesn't work?

Reply:
Returning nil/nothing will result in the `require` function to return `true`, which I didn't know when writing, and `events.lua` conducts `if not queue` checks because I wrongly expected `require` to return nil.

Reply:
What's the ox_lib change then?

Reply:
Previous ox_lib turned any falsy return values from modules to `true`, when only nil values were supposed to become `true`. As mentioned in the description of this PR I bumped to a version that includes the fix for that, so that returning `false` works correctly.